main.py

```diff
@@ -63,9 +63,25 @@ def create_map_fig(df, DAYS):
             t=75,
             b=75,
         ),
+        title={
```

Comment:
This is not a blocker, but one thing to try and avoid is mixing styles in dictionaries in the same scope of code. It's a code style thing, but still important to note.

i.e. you are using the `{}` syntax for a dictionary here, but using the `dict()` syntax above, in the same `px. scatter_map` usage.